reprospect/test/sass/instruction/half.py

```diff
@@ -93,24 +102,24 @@ class Fp16FusedMulAddMatcher(PatternMatcher):
     TEMPLATE: typing.Final[str] = f"{PatternBuilder.opcode_mods('HFMA2', modifiers = ('?MMA',))} {Register.dst()}, {{op1}}, {{op2}}, {{op3}}(?:, {{op4}})?"
 
     PATTERN_ANY: typing.Final[regex.Pattern[str]] = regex.compile(TEMPLATE.format(
-        op1=Fp16.any_operand(),
-        op2=Fp16.any_operand(),
-        op3=Fp16.any_operand(),
-        op4=Fp16.any_operand(),
+        op1=Fp16.operand(half_sel=None),
+        op2=Fp16.operand(half_sel=None),
+        op3=Fp16.operand(half_sel=None),
+        op4=Fp16.operand(half_sel=None),
```

Comment:
Either be explicit on all or on none.
In this case, I think the intent is really to have "any" operand", so probably best to given math=None and immediate=None